addons/mail/tests/test_parse_inline_template.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import BaseCase
+from odoo.tools import parse_inline_template
+
+class TestParseInlineTemplate(BaseCase):
```

Comment:
I bet you could add more test cases (and probably move to base as this does not really depends on mail). Purpose is to keep mail-related tests in mail, tools tests should be in base. That way tests are run with their core code definition.